tests/integration/telemetry/stackdriver/fake/stackdriver_test.go

```diff
@@ -0,0 +1,40 @@
+// +build integ
+// Copyright Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package google
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	sd "istio.io/istio/tests/integration/telemetry/stackdriver"
+)
+
+func TestMain(m *testing.M) {
+	s := framework.NewSuite(m)
+	sd.TestMainSetup(s, false /* useRealSD */)
```

Comment:
@nmittler did you want this to be framework argument controlled, or is this OK?

Reply:
Yeah, maybe a framework flag?

Reply:
It is great to have a framework flag for this. I initially thought this would be somehow controversial. Add a telemetry.go file to top level framework package which includes all telemetry flags. PTAL, thanks!